src/odemis/gui/util/img.py

```diff
@@ -2621,37 +2621,38 @@ def insert_tile_to_image(tile, ovv):
     return ovv
 
 
-def merge_screen(im, background):
+def merge_screen(ima, imb):
     """ 
-    Merges two images (im and background) into one using the "screen" operator:
+    Merges two images into one using the "screen" operator:
     f(xA,xB) = xA + xB − xA·xB (with values between 0 and 1, with each channel independent)
-    im, background: DataArray with im.shape = background.shape (either RGB or RGBA)
-    returns RGBA DataArray (of the same YX as im, but with always depth=4)
+    ima, imb: DataArray with ima.shape = imb.shape (either RGB or RGBA)
+    returns RGBA DataArray (of the same YX as ima, but with always depth=4)
```

Comment:
> but with always depth=4
Maybe changing this to: 'but with always RGBA format' that is slightly clearer than depth 4 I think.

Reply:
I respectfully disagree. We already just stated that it returns  a RGBA array. So it's not useful to indicate that it's RGBA to explain it. Maybe I could write "of shape YXC, with YX the same as ima, and C=4)".